### ISSUE TITLE
[Bug?]: Issues upgrading from 4.3.1 to 4.4.1

### ISSUE THREAD

**Issue by amicharski** [redwoodjs/redwood#7910]
### What's not working?

I follow the exact commands it tells me to do. Here is a copy/paste of my shell:
```
PS A:\Node\profrec> yarn rw dev
The currently configured ports for the development server are unavailable. Suggested changes to your ports, which can be changed in redwood.toml, are:
  - API to use port 8912 instead of your currently configured 8911

Cannot run the development server until your configured ports are changed or become available.

   ╭ Redwood Update available 🎉 ────────────────────────────────────────╮
   │ New updates to Redwood are available via `yarn rw upgrade`          │
   │                                                                     │
   │ ❖  latest:                                                          │
   │     v4.3.1 -> v4.4.1                                                │
   │                                                                     │
   │ See release notes at: https://github.com/redwoodjs/redwood/releases │
   ╰─────────────────────────────────────────────────────────────────────╯

PS A:\Node\profrec> yarn rw upgrade
✔ Checking latest version
✔ Updating your project package.json(s)
  ✔ Updating A:\Node\profrec\package.json
  ✔ Updating A:\Node\profrec\api\package.json
  ✔ Updating A:\Node\profrec\web\package.json
✖ Could not finish installation. Please run `yarn install --force`, before continuing
◼ Refreshing the Prisma client
◼ De-duplicating dependencies
◼ One more thing..
Could not finish installation. Please run `yarn install --force`, before continuing
PS A:\Node\profrec> yarn install --force
Unknown Syntax Error: Unsupported option name ("--force").

$ yarn install [--json] [--immutable] [--immutable-cache] [--check-cache] [--inline-builds] [--mode #0]
PS A:\Node\profrec> yarn install --force
Unknown Syntax Error: Unsupported option name ("--force").

$ yarn install [--json] [--immutable] [--immutable-cache] [--check-cache] [--inline-builds] [--mode #0]
PS A:\Node\profrec> 
```
Also when I run `yarn rw info`, I get this error:
```
PS A:\Node\profrec> yarn rw info
Usage Error: Couldn't find a script named "rw".

$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...
```

### How do we reproduce the bug?

1) Be on 4.3.1
2) Run `yarn rw dev`
3) Follow the upgrade instructions

### What's your environment? (If it applies)

```shell
I can't run `yarn rw info` without an error, so I'll show you what the output was yesterday which was the last time I ran it:

System:
    OS: Windows 10 10.0.19044
  Binaries:
    Node: 16.19.0 - ~\AppData\Local\Temp\xfs-880a4f99\node.CMD
    Yarn: 3.4.1 - ~\AppData\Local\Temp\xfs-880a4f99\yarn.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (111.0.1661.51)
  npmPackages:
    @redwoodjs/auth-dbauth-setup: 4.3.1 => 4.3.1
    @redwoodjs/core: 4.3.1 => 4.3.1
  Database:
    PostgreSQL 15.2 (remote server on local intranet)
```

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by thedavidprice** [comment]
Hi @amicharski Thank you for asking about this.

I believe you have a couple things going on:
- dev server (or something else) in the background running on 8911 (or 8910)
- yarn install busted (who knows, it happens 🤷‍♂️), but then the prompt gave you a yarn v1 command deprecated in v3

If you haven't resolved this yet, there are a few ways forward:
- make sure you've shut down dev server; check running Node processes and kill anything on 8910 and 8911 (worst case scenario, restart computer)
- manually confirm you're at the latest version in the 3 package.json files; save
```
  ✔ Updating A:\Node\profrec\package.json
  ✔ Updating A:\Node\profrec\api\package.json
  ✔ Updating A:\Node\profrec\web\package.json
```
- personally, I would delete all node_modules and reinstall (you can do this with `git clean -xfd -e .env` but WARNING that this can wipe out .env files and others not added to Git versioning... read up on this command before you use it)
- `yarn`

That should do it!

## Fix the Incorrect Prompt
This line is incorrect:
```
✖ Could not finish installation. Please run `yarn install --force`, before continuing
```

It should just be `yarn install`. 

Would you like to open a PR to help with this?

**Comment by amicharski** [comment]
I would love to help, but can you help me through the process because I never did a pr before on github?

**Comment by pantheredeye** [comment]
Hey @amicharski we would love for you to contribute! This document will help you get everything set up: https://redwoodjs.com/docs/contributing-walkthrough#gitpod-browser-based-development

There is a lot to read through, but please do not be discouraged! Some of it is just the explanation of tools, workflow and terminology, and there is some OS specific instructions. However, this is one of the best resources I have found in any project for getting set up to contribute.  Please do not hesitate to ask for help if you get stuck - here, or [on our forums](https://community.redwoodjs.com/), or [our discord](https://discord.gg/redwoodjs).

**Comment by thedavidprice** [comment]
We have a plethora of content to help: https://redwoodjs.com/docs/contributing

The video walkthrough is likely the most helpful in this case.

Do know this change will be very lightweight because it's just text output from a CLI command. Here's what I'd do:
- fork repo
- search (in GH) for `Could not finish installation. Please run `yarn install --force`, before continuing`
- edit and commit
- open PR

Zero pressure. But if you want to try and can turn it around quickly, please do!

**Comment by thedavidprice** [comment]
I went ahead and updated the copy. This will be included in v4.5
